### PR TITLE
in GDScript missing closing codeblock tag. removed trailing whitespace

### PR DESCRIPTION
**Pull request by willt**
preload was missing a closing codeblock tag
asin had some trailing whitespace in its codeblock

### CODE REVIEW

**Comment by NathanLovato** [comment]
All good

**Review by NathanLovato — APPROVED** [review]
All good

File: doc/base/classes.xml [hunk 1]
```diff
@@ -79,7 +79,7 @@
 				Returns the arc sine of 's' in radians. Use to get the angle of sine 's'.
 				[codeblock]
 				# s is 0.523599 or 30 degrees if converted with rad2deg(s)
-				s = asin(0.5)				
+				s = asin(0.5)
```
Comment (djrm) [inline doc/base/classes.xml]: shouldn't this be
`var s = asin(0.5)`

Comment (willt) [reply doc/base/classes.xml]: no var is intentionally left out to keep things simple. s could be declared elsewhere.

**Comment by akien-mga** [comment]
Superseded by #11227.

**Pull request closed without merging**